ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -349,7 +402,7 @@ static TensorMemoryLayout select_shard_spec(
 
     // Prefer block sharding over height sharding but make sure that we got at least
     // some blocking on width dimension as well.
-    if (cc_height > max_cc || (cc_height == max_cc && cc_height <= compute_grid_size.x)) {
+    if ((cc_height > max_cc && max_cc < 48) || (cc_height == max_cc && cc_height <= compute_grid_size.x)) {
```

Comment:
Maybe put 48 into a variable with a descriptive name. I don't know the importance of 48 and why it is chosen.

Reply:
done